misc/services/mobile-webui/mobile-webui-frontend/src/components/dialogs/GetQuantityDialog.jsx

```diff
@@ -172,6 +172,7 @@ const GetQuantityDialog = ({
         bestBeforeDate: qrCode.bestBeforeDate,
         lotNo: qrCode.lotNo,
         productNo: qrCode.productNo,
+        ean13ProductCode: qrCode.ean13ProductCode,
```

Comment:
hmmm, from where this is coming? in ean13.js/parseEAN13CodeString I see no ean13ProductCode.
i think it's not needed

Reply:
in case of EAN13, IMHO the qrCode.productNo shall already contained the correctly parsed ean13 product no.